src/main/java/uk/nhs/hee/tis/trainee/credentials/config/GatewayProperties.java

```diff
@@ -64,4 +65,14 @@ public record TokenProperties(String audience, String issuer, String signingKey)
 
     }
   }
+
+  /**
+   * A representation of the gateway verification properties.
+   *
+   * @param authorizeEndpoint The gateway's connect authorize endpoint URI.
+   */
+  @ConfigurationProperties(prefix = "application.gateway.verification")
+  public record VerificationProperties(String authorizeEndpoint) {
```

Comment:
Given we include the `redirectUri` parameter for the `IssuingProperties`, is there a particular reason to omit it here?